### ISSUE TITLE
Cannot find module 'webpack'

### ISSUE THREAD

**Issue by davej** [Kiyozz/electron-esbuild#3]
Tried running:
```
npx create-electron-esbuild-app --name my-app --package-manager npm --template react-typescript
cd my-app
npm run dev
```

And got the error:
```
> my-app@ dev /Users/dave/code/my-app
> electron-esbuild dev

internal/modules/cjs/loader.js:895
  throw err;
  ^

Error: Cannot find module 'webpack'
Require stack:
- /Users/dave/code/my-app/node_modules/webpack-dev-server/lib/Server.js
- /Users/dave/code/my-app/node_modules/electron-esbuild/dist/builder/webpack.js
- /Users/dave/code/my-app/node_modules/electron-esbuild/dist/builder.js
- /Users/dave/code/my-app/node_modules/electron-esbuild/dist/commands/build.js
- /Users/dave/code/my-app/node_modules/electron-esbuild/dist/commands/index.js
- /Users/dave/code/my-app/node_modules/electron-esbuild/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:892:15)
    at Function.Module._load (internal/modules/cjs/loader.js:742:27)
    at Module.require (internal/modules/cjs/loader.js:964:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/dave/code/my-app/node_modules/webpack-dev-server/lib/Server.js:23:17)
    at Module._compile (internal/modules/cjs/loader.js:1075:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1096:10)
    at Module.load (internal/modules/cjs/loader.js:940:32)
    at Function.Module._load (internal/modules/cjs/loader.js:781:14)
    at Module.require (internal/modules/cjs/loader.js:964:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/dave/code/my-app/node_modules/webpack-dev-server/lib/Server.js',
    '/Users/dave/code/my-app/node_modules/electron-esbuild/dist/builder/webpack.js',
    '/Users/dave/code/my-app/node_modules/electron-esbuild/dist/builder.js',
    '/Users/dave/code/my-app/node_modules/electron-esbuild/dist/commands/build.js',
    '/Users/dave/code/my-app/node_modules/electron-esbuild/dist/commands/index.js',
    '/Users/dave/code/my-app/node_modules/electron-esbuild/dist/index.js'
  ]
}
```

Using Node 14.9.0.

**Comment by Kiyozz** [comment]
Hum, I can't reproduce. But there is indeed a problem with current version of create-electron-esbuild-app in templates with latest version of electron-esbuild.

**Comment by Kiyozz** [comment]
I can reproduce with `pnpm`. electron-esbuild use webpack-dev-server for webpack config. I'll make a fix for this.

**Comment by Kiyozz** [comment]
It should be fixed with create-electron-esbuild-app@1.2.1 and electron-esbuild@1.2.4.

Thanks for reporting :)
